meshapp/src/ProfileCardCarousel/Swiper.tsx

```diff
@@ -0,0 +1,219 @@
+        "BARK BARK WOOF WOOF RUFF RUFF BARK WOOF RUFF RUFF WOOF GRRR BARK BARK RUFF RUFF GRR WOOF WOOF GRRRR MEOW GRRRR MEOW MEOW MEWWWWW WOOOF WOOF GRRR RUFF MEWO MEWO",
+      interests: ["insider trading", "cats", "meow", "el gato", "kitty"],
+      details: [
+        "bureaucat in training",
+        "B.S in Milk Peddling",
+        "Looking for a Mentor",
+      ],
+    },
+    {
+      image:
+        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTyFmpv3VDfx13aPqlY__oyW4Xpmi7VoEYo3cMcejNL9A&usqp=CAU&ec=48665698",
+      name: "Alex Mrownes",
+      job: "propagandist",
+      biography: "MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEEO MEW",
+      interests: ["cats", "meow", "fearmongering", "el gato", "kitty"],
+      details: [
+        "cat propagandist in training",
+        "B.S in Milk Stockpiling",
+        "Looking for a Mentor",
+      ],
+    },
+  ];
+
+  //keeps track of the current card we're on
+  const [position, positionSet] = useState(0);
+
+  //moves to the next card if possible
+  const onRight = () => {
+    if (position < profiles.length - 1) {
+      positionSet(position + 1);
+    }
+  };
+
+  //moves to previous card if possible
+  const onLeft = () => {
+    if (position > 0) {
+      positionSet(position - 1);
+    }
+  };
+
+  //allows mouse/touch swiping to traverse through cards
+  const handlers = useSwipeable({
+    delta: 0,
+    onSwipedLeft: () => onRight(),
+    onSwipedRight: () => onLeft(),
+    preventScrollOnSwipe: true,
+    trackMouse: true,
+  });
+
+  //keeps track of window width
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    //keeps track of screen width
+    function setWidth() {
+      setWindowWidth(window.innerWidth);
+    }
+
+    //adds keyboard event listeners, goes to the right if arrow right or 'd' key, goes to left if arrow left or 'a' key
+    function keyPress(e: KeyboardEvent) {
+      let { key } = e;
+      key = key.toLowerCase();
+      if (key === "arrowright" || key === "d") {
+        onRight();
+      } else if (key === "arrowleft" || key === "a") {
+        onLeft();
+      }
+    }
+    window.addEventListener("resize", setWidth);
+    window.addEventListener("keydown", keyPress);
+
+    return () => {
+      window.removeEventListener("resize", setWidth);
+      window.removeEventListener("keydown", keyPress);
+    };
+  });
+
+  //array of all the user profile cards
+  let divs = profiles.map(
+    ({ image, name, job, biography, interests, details }, index: number) => {
+      //controls which card shows up on top of the stack
+      let onTop: number = 0;
+      //controls the positioning of the cards depending on what index they are from the current position
+      //if window width is lower than 700 it will stack the cards closer together
+      let leftPos: string =
+        windowWidth > 700
+          ? `${(index - position) * 15 + 14}vw`
+          : `${(index - position) * 10 + 4.5}vw`;
```

Comment:
There are too many magic-numbers here. I suggest converting these into constants to make your code more readable and easier to update later (i.e. `private static readonly MAX_WINDOW_WIDTH : number = 700;`).

Reply:
I have converted them to constants